### PR TITLE
Fix brackets display

### PR DESCRIPTION
**Pull request by jianan1104**
# Overview

Fix brackets display, replace `\left(` to `\mathopen{}\left(` and `\right)` to `\mathclose{}\right)`

# Details

Modify function `def _wrap_operand` in the `src/latexify/function_codegen.py` line by `422`, Change the original brackets string to fix the display.

- before
image

- after
截圖 2022-11-16 上午1 21 26

# References

#90

### CODE REVIEW

**Comment by odashi** [comment]
@jianan1104 Thanks! You also need to fix all tests that relies on this change. Running `./checks.sh` could check if the test passes or not.

**Review by odashi — COMMENTED** [review]
I think there are other brackets in `function_codegen.py`, and it would be great if we fixed all of them.

**Comment by jianan1104** [comment]
> @jianan1104 Thanks! You also need to fix all tests that are related to this change. Running `./checks.sh` could check if the test passes or not.

Thank you for reviewing. No problem. I'll fix it asap.

**Comment by jianan1104** [comment]
@odashi Hi, odashi. just fix and test the code, please check it.

File: src/integration_tests/regression_test.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 from __future__ import annotations
 
-from collections.abc import Callable
 import math
+from collections.abc import Callable
```
Comment (odashi) [inline src/integration_tests/regression_test.py:6]: There are some changes about only the import order, which are not related to the essential point of this pull request. Could you revert these changes to prevent breaking the blame history?

(Btw, I think I need to introduce some static checker (e.g., isort) to constrain the style around imports.)

**Review by odashi — COMMENTED** [review]
The logic looks fine, thanks!

It looks some lines exceed the the length limit (88 characters) though I didn't realize why Black didn't warn about them. Could you fix them as well?

**Comment by odashi** [comment]
It looks the CI accidentally passes though it detected some errors:
```
Run python -m pflake8 src
src/integration_tests/regression_test.py:91:89: E501 line too long ([11](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:12)4 > 88 characters)
src/integration_tests/regression_test.py:99:89: E501 line too long (115 > 88 characters)
src/integration_tests/regression_test.py:227:89: E501 line too long (132 > 88 characters)
src/latexify/__init__.py:7:1: E722 do not use bare 'except'
src/latexify/codegen/function_codegen_test.py:59:89: E501 line too long (107 > 88 characters)
src/latexify/codegen/function_codegen_test.py:75:89: E501 line too long (108 > 88 characters)
src/latexify/codegen/function_codegen_test.py:104:89: E501 line too long (107 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[12](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:13)0:89: E501 line too long (108 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[14](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:15)5:89: E501 line too long (103 > 88 characters)
src/latexify/codegen/function_codegen_test.py:149:89: E501 line too long (105 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[15](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:16)3:89: E501 line too long (115 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[17](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:18)7:89: E501 line too long (125 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[19](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:20)4:89: E501 line too long (91 > 88 characters)
src/latexify/codegen/function_codegen_test.py:198:89: E501 line too long (109 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[20](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:21)3:89: E501 line too long (93 > 88 characters)
src/latexify/codegen/function_codegen_test.py:[22](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:23)3:89: E501 line too long (119 > 88 characters)
src/latexify/codegen/function_codegen_test.py:2[28](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:29):89: E501 line too long (114 > 88 characters)
src/latexify/codegen/function_codegen_test.py:229:89: E501 line too long (111 > 88 characters)
```

File: checks.sh [hunk 2]
```diff
@@ -3,4 +3,4 @@ set -eoux pipefail
 
 python -m pytest src -vv
 python -m black --check src
-python -m pflake8 src
+python -m flake8 src
```
[the remark this answers is not in the document]
Comment (jianan1104) [reply checks.sh:6]: Okay. sorry for this change. I have no experience with pflake8(flake8 for pyproject), so I just edit it by intuition.

File: src/integration_tests/regression_test.py (hunk 1 above)
Comment (jianan1104) [reply src/integration_tests/regression_test.py:6]: No problem.

**Comment by jianan1104** [comment]
> It looks the CI accidentally passes though it detected some errors:
> 
> ```
> Run python -m pflake8 src
> src/integration_tests/regression_test.py:91:89: E501 line too long ([11](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:12)4 > 88 characters)
> src/integration_tests/regression_test.py:99:89: E501 line too long (115 > 88 characters)
> src/integration_tests/regression_test.py:227:89: E501 line too long (132 > 88 characters)
> src/latexify/__init__.py:7:1: E722 do not use bare 'except'
> src/latexify/codegen/function_codegen_test.py:59:89: E501 line too long (107 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:75:89: E501 line too long (108 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:104:89: E501 line too long (107 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[12](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:13)0:89: E501 line too long (108 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[14](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:15)5:89: E501 line too long (103 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:149:89: E501 line too long (105 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[15](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:16)3:89: E501 line too long (115 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[17](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:18)7:89: E501 line too long (125 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[19](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:20)4:89: E501 line too long (91 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:198:89: E501 line too long (109 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[20](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:21)3:89: E501 line too long (93 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:[22](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:23)3:89: E501 line too long (119 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:2[28](https://github.com/google/latexify_py/actions/runs/3476330689/jobs/5811657602#step:5:29):89: E501 line too long (114 > 88 characters)
> src/latexify/codegen/function_codegen_test.py:229:89: E501 line too long (111 > 88 characters)
> ```

I do not know if any lint tool can fix automatically? or just edit line by line?

**Comment by odashi** [comment]
> I do not know if any lint tool can fix automatically? or just edit line by line?

Length limit error is usually resolved by running Black, but I guess long string literals couldn't be handled correctly. So (unfortunately) some errors need to be fixed manually.

Btw, I also noticed that the list above contains some existing errors in `main`. I will fix it first ASAP.

It looks the accident around flake8 was reported recently:
https://github.com/csachs/pyproject-flake8/issues/19

**Comment by odashi** [comment]
@jianan1104 Could you merge the `main` so that the flake8 errors kill the CI appropriately?

**Comment by jianan1104** [comment]
> @jianan1104 I fixed CIs. Could you merge the `main` into this pull request so that the flake8 errors kill the CI appropriately?

Sure.

**Comment by jianan1104** [comment]
> > I do not know if any lint tool can fix automatically? or just edit line by line?
> 
> Length limit error is usually resolved by running Black, but I guess long string literals couldn't be handled correctly. So (unfortunately) some errors need to be fixed manually.
> 
> Btw, I also noticed that the list above contains some existing errors in `main`. I will fix it first ASAP.
> 
> It looks the accident around flake8 was reported recently: [csachs/pyproject-flake8#19](https://github.com/csachs/pyproject-flake8/issues/19)

Okay. I'll fix it asap!!

File: src/integration_tests/regression_test.py (hunk 1 above)
Comment (jianan1104) [reply src/integration_tests/regression_test.py]: Done

**Review by odashi — APPROVED** [review]
Really great work, thanks!

**Comment by odashi** [comment]
Fixes #90

**Comment by jianan1104** [comment]
> Really great work, thanks!

Glad to be a contributor.
